nengo/utils/tests/test_least_squares_solvers.py

```diff
@@ -13,7 +13,9 @@
 )
 
 
-def run_solver(solver, m=30, n=20, d=1, cond=10, sys_rng=np.random, **kwargs):
+def run_solver(
+    solver, m=30, n=20, d=1, cond=10, sys_rng=np.random, measure="x", **kwargs
```

Comment:
New arguments that modify what the function does (especially what it returns) is an anti-pattern IMO... I think in this case I would just `return x, x2, A` and calculate `y` and `y2` in `test_conjgrad`.

Reply:
Hah, I had something more like that before (I just returned `x, x2, y, y2`), but then it seemed silly to modify all the other tests to `x, x2, _, _ = run_solver` when nothing had really changed about them. I can change it back, though...